Harvest.Web/Harvest.Web.csproj

```diff
@@ -45,6 +45,16 @@
     <None Include="$(SpaRoot)**" Exclude="$(SpaRoot)node_modules\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="ClientApp\public\error.png" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="ClientApp\public\error.png">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
```

Comment:
You don't need this -- see for example that this page references `/media/ucdavis_white_logo.svg` just fine and it's not in the csproj.  CRA will build and anything in the ClientApp/public folder will be made available at the root, nothing needed in csproj